app/Http/Controllers/Multiplayer/RoomsController.php

```diff
@@ -18,6 +18,19 @@ public function __construct()
         $this->middleware('require-scopes:public', ['only' => ['index', 'leaderboard', 'show']]);
     }
 
+    /**
+     * Get Multiplayer Rooms
+     *
+     * Returns a list of multiplayer rooms.
+     *
+     * ---
+     *
+     * @queryParam limit Maximum number of results. No-example
+     * @queryParam mode Filter mode; `all`, `ended`, `participated`, `owned`. Defaults to active rooms only if not sepcified. No-example
```

Comment:
can just pretend `active` param for the default (and maybe add that as part of the switch)

Reply:
wait, that's why it was `default` in the first place 🙈